src/consumer.ts

```diff
@@ -248,7 +262,10 @@ export class Consumer extends EventEmitter {
 
   private poll(): void {
     if (this.stopped) {
-      this.emit('stopped');
+      this.emit(
+        'error',
+        new Error('Poll was called while consumer was stopped')
+      );
```

Comment:
Tests are currently failing because of this error, need to think on how best to validate that the poll has be stopped when stop was called, that this error is not called is probably valid, which would mean that the tests are failing correctly, currently.

Reply:
The code isn't actually cancelling the timeout as well, still getting the same logs

Reply:
So now that I have thought about it, the problem makes a lot of sense, the timeout is not created until a response from SQS is received, so if we wait 25s ish, and then stop, start the consumer, the timeout will have been created and then cleared when stopped, so the code is working in this clearTimeout implementation, however, not for the case where you immediately start, stop, start.

Reply:
I think I've resolved this by simply adding a check for if the timeout already exists before trying to set it, if it does, clear it.